### PR TITLE
1.0.0 release notes

### PR DESCRIPTION
**Pull request by amontanez24**
view release notes here: https://github.com/sdv-dev/SDV/blob/1.0.0-release-notes/HISTORY.md

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -1,5 +1,183 @@
...
+* `TVAE` -> `TVAESynthesizer`
+* `CopulaGan` -> `CopulaGANSynthesizer`
+* `PAR` -> `PARSynthesizer`
+* `HMA1` -> `HMASynthesizer`
+
+In `SDV` 1.0, synthesizers are classes that take in metadata and handle data preprocessing, model training and model sampling. This is similar to the previous `BaseTabularModel` in `SDV` <1.0.
+
+`Synthesizers` in `SDV` 1.0 define a clear workflow for generating synthetic data.
+1. Synthesizers are initialized with a metadata class.
+2. They can then be used to transform the data and apply constraints using the `synthesizer.preprocess()` method. This step also validate that the data matches the provided metadata to avoid errors in fitting or sampling.
```
Comment (frances-h) [inline HISTORY.md:31]: Typo: "This step also validate" -> "This step also validates"

File: HISTORY.md [hunk 2]
```diff
@@ -1,5 +1,183 @@
...
+3. The processed data can then be fed into the underlying machine learning model using `synthesizer.fit_processed_data()`. (Alternatively, data can be preprocessed and fit to the model using `synthesizer.fit()`.)
+4. Data can then be sampled using `synthesizer.sample()`.
+
+Each synthesizer class also provides a series of methods to help users customize the transformations their data goes through. Read more about that [here](https://docs.sdv.dev/sdv/single-table-data/modeling/synthetic-data-workflow/transform-and-anonymize).
+
+Notice that the preprocessing and model fitting steps can now be separated. This can be helpful if preprocessing is time consuming or if the data has been processed externally.
+
+Another major addition is control over randomization. In `SDV` <1.0, users could set a seed to control the randomization for only some columns. In `SDV` 1.0, randomization is controlled for all columns. Every new call to sample generates new data, but the synthesizers seed can be reset to the original state using `synthesizer.reset_randomization()`, enabling reproducibility.
```
Comment (frances-h) [inline HISTORY.md:39]: typo: "but the synthesizers seed" ->  "but the synthesizer's seed"

File: HISTORY.md [hunk 3]
```diff
@@ -1,5 +1,183 @@
 # Release Notes
 
+## 1.0.0 - 2023-03-28
+
+This is a major release that introduces a new API to the `SDV` aimed at streamlining the process of synthetic data generation! To achieve this, this release includes the addition of several large features.
+
```
Comment (npatki) [inline HISTORY.md:6]: I think some sub-headers may actually be really nice here.

Eg. This section can be 
```
### Metadata
```

File: HISTORY.md [hunk 4]
```diff
@@ -1,5 +1,183 @@
...
+Some of the most notable additions are the new `SingleTableMetadata` and `MultiTableMetadata` classes. These classes enable a number of features that make it easier to synthesize your data correctly such as:
+
+* Automatic data detection - Calling `metadata.detect_from_dataframe()` or `metadata.detect_from_csv()` will populate the metadata autonomously with values it thinks represent the data.
+* Easy updating - Once an instance of the metadata is created, values can be easily updated using a number of methods defined in the API. For more info, view the [docs](https://docs.sdv.dev/sdv/single-table-data/data-preparation/single-table-metadata-api).
+* Metadata validation - Calling `metadata.validate()` will return a report of any invalid definitions in the metadata specification.
+* Upgrading - Users with the previous metadata format can easily update to the new specification using the `upgrade_metadata()` method.
+* Saving and loading - The metadata itself can easily be saved to a json file and loaded back up later.
+
```
Comment (npatki) [inline HISTORY.md:14]: Sub-header

```
### Class and Module Names
```

File: HISTORY.md [hunk 5]
```diff
@@ -1,5 +1,183 @@
...
+* `BaseTabularModel` -> `BaseSingleTableSynthesizer`
+* `GaussianCopula` -> `GaussianCopulaSynthesizer`
+* `CTGAN` -> `CTGANSynthesizer`
+* `TVAE` -> `TVAESynthesizer`
+* `CopulaGan` -> `CopulaGANSynthesizer`
+* `PAR` -> `PARSynthesizer`
+* `HMA1` -> `HMASynthesizer`
+
+In `SDV` 1.0, synthesizers are classes that take in metadata and handle data preprocessing, model training and model sampling. This is similar to the previous `BaseTabularModel` in `SDV` <1.0.
+
```
Comment (npatki) [inline HISTORY.md:28]: Sub-header

```
### Synthetic Data Workflow
```

File: HISTORY.md [hunk 6]
```diff
@@ -1,5 +1,183 @@
...
+`Synthesizers` in `SDV` 1.0 define a clear workflow for generating synthetic data.
+1. Synthesizers are initialized with a metadata class.
+2. They can then be used to transform the data and apply constraints using the `synthesizer.preprocess()` method. This step also validate that the data matches the provided metadata to avoid errors in fitting or sampling.
+3. The processed data can then be fed into the underlying machine learning model using `synthesizer.fit_processed_data()`. (Alternatively, data can be preprocessed and fit to the model using `synthesizer.fit()`.)
+4. Data can then be sampled using `synthesizer.sample()`.
+
+Each synthesizer class also provides a series of methods to help users customize the transformations their data goes through. Read more about that [here](https://docs.sdv.dev/sdv/single-table-data/modeling/synthetic-data-workflow/transform-and-anonymize).
+
+Notice that the preprocessing and model fitting steps can now be separated. This can be helpful if preprocessing is time consuming or if the data has been processed externally.
+
```
Comment (npatki) [inline HISTORY.md:38]: Sub-Header

```
### Other Highly Requested Features
```

File: HISTORY.md [hunk 7]
```diff
@@ -1,5 +1,183 @@
...
+PII handling is improved by the following features:
+* Primary keys can be set to natural sdtypes (eg. SSN, email, name). Previously they could only be numerical or text.
+* The `PseudoAnonymizedFaker` can be used to provide consistent mapping to PII columns. The mapping itself can be accessed by viewing the transformers for the column using `synthesizer.get_transformers()`.
+* A bug causing PII columns to slow down modeling is patched.
+
+Finally, the synthetic data can now be easily evaluated using the `evaluate_quality()` and `run_diagnostic()` methods. The data can be compared visually to the actual data using the `get_column_plot()` and `get_column_pair_plot()` methods. For more info on how to visualize or interpret the synthetic data evaluation, read the docs [here](https://docs.sdv.dev/sdv/single-table-data/evaluation).
+
```
Comment (npatki) [inline HISTORY.md:47]: Then maybe we can make this a new major section

```
## Details
```

Everything you have underneath it is then sub-headers of it.

**Review by pvk-developer — APPROVED** [review]
LGTM :shipit: 🚀

File: HISTORY.md [hunk 8]
```diff
@@ -1,5 +1,199 @@
...
+* In `upgrade_metadata`, return the object instead of writing it to a JSON file - Issue [#1319](https://github.com/sdv-dev/SDV/issues/1319) by @frances-h
+* In `upgrade_metadata` index primary keys should be converted to `text` - Issue [#1318](https://github.com/sdv-dev/SDV/issues/1318) by @amontanez24
+* Add `load_from_dict` to SingleTableMetadata and MultiTableMetadata - Issue [#1314](https://github.com/sdv-dev/SDV/issues/1314) by @amontanez24
+* Throw a `SynthesizerInputError` if `FixedCombinations` constraint is applied to a column that is not `boolean` or `categorical` - Issue [#1306](https://github.com/sdv-dev/SDV/issues/1306) by @frances-h
+* Add integration tests for 'Adding Constraints' demo - Issue [#1280](https://github.com/sdv-dev/SDV/issues/1280) by @pvk-developer
+* Add integration tests to the 'Use Your Own Data' demo - Issue [#1278](https://github.com/sdv-dev/SDV/issues/1278) by @frances-h
+* Add integration tests for 'Synthesize Multi Tables' demo - Issue [#1277](https://github.com/sdv-dev/SDV/issues/1277) by @pvk-developer
+* Add integration tests for 'Synthesize a Table' demo - Issue [#1276](https://github.com/sdv-dev/SDV/issues/1276) by @frances-h
```
[the remark this answers is not in the document]
Comment (amontanez24) [reply HISTORY.md:73]: I wrote a script to generate these. If the issue was tagged as a feature request, that got preference over any other tags. These are all tagged with both

[the remark this answers is not in the document]
Comment (amontanez24) [reply HISTORY.md:73]: Although they make more sense as internal. I can remove the feature request tags and move them

[the remark this answers is not in the document]
Comment (frances-h) [reply HISTORY.md]: Yeah I'm not sure it matters either way, was mostly just curious as to why #1295 was separate.

**Pull request merged**